Makefile

```diff
@@ -20,7 +20,7 @@ VENV_ACTIVATE := .venv/bin/activate
 ${VENV_ACTIVATE}:
 	python3 -m pip install --user virtualenv
```

Comment:
If the intent is to just use `venv` instead of `virtualenv` then this line can be removed.

Reply:
Yes, right, removed. Thanks for noticing.